docs/getting_started.md

```diff
@@ -27,14 +27,47 @@ kubectl create clusterrolebinding YOURNAME-cluster-admin-binding --clusterrole=c
 
 Download the latest Argo CD version from [https://github.com/argoproj/argo-cd/releases/latest](https://github.com/argoproj/argo-cd/releases/latest).
 
-Also available in Mac Homebrew:
+### Linux
+
+You can view the lastest version of Argo CD at the link above or run the following command to grab the version:
+
+```bash
+curl --silent "https://api.github.com/repos/argoproj/argo-cd/releases/latest" | grep '"tag_name"' | sed -E 's/.*"([^"]+)".*/\1/'
+```
+
+Replace `REPLACE_VERSION` in the command below with the version of Argo CD you would like to download:
```

Comment:
For getting started we’d like to keep it simple. It think pointing people to where to download the latest Linux binary would be enough.

Reply:
Sure, were you thinking something like I just pushed or removing this all together from getting_started.md and putting this somewhere else?

Reply:
Maybe a separate page?

Reply:
Yes, I think that makes sense. My apologies. I thought to put them in getting started since it's the only place I found these instructions. I can draft a new doc and perhaps link to it getting started in case more detail is needed.